src/cloudant/result.py

```diff
@@ -327,7 +328,7 @@ def _handle_result_by_key_slice(self, key_slice):
     def __iter__(self):
         """
         Provides iteration support, primarily for large data collections.
-        The iterator uses the ``skip`` and ``limit`` options to consume
+        The iterator uses the ``startkey`` and ``limit`` options to consume
```

Comment:
and ``startkey_docid``

Reply:
ah ok, I've left it back on purpose viewing it as an "hidden feature" but I'll amend docs